packages/lib/src/spatial-navigation/components/virtualizedList/SpatialNavigationVirtualizedListWithScroll.tsx

```diff
@@ -47,17 +52,108 @@ export type SpatialNavigationVirtualizedListWithScrollProps<T> = Omit<
   'currentlyFocusedItemIndex'
 >;
 
+export type PointerScrollProps = {
+  descendingArrow?: React.ReactElement;
+  descendingArrowContainerStyle?: ViewStyle;
+  ascendingArrow?: React.ReactElement;
+  ascendingArrowContainerStyle?: ViewStyle;
+  scrollInterval?: number;
+};
+
 /**
  * This component wraps every item of a virtualizedList in a scroll handling context.
  */
 export const SpatialNavigationVirtualizedListWithScroll = typedMemo(
-  <T extends ItemWithIndex>(props: SpatialNavigationVirtualizedListWithScrollProps<T>) => {
-    const { renderItem } = props;
+  <T extends ItemWithIndex>(
+    props: SpatialNavigationVirtualizedListWithScrollProps<T> & PointerScrollProps,
+  ) => {
+    const {
+      data,
+      renderItem,
+      descendingArrow: descendingArrow,
+      ascendingArrow: ascendingArrow,
+      descendingArrowContainerStyle: descendingArrowContainerStyle,
+      ascendingArrowContainerStyle: ascendingArrowContainerStyle,
+      scrollInterval = 100,
+    } = props;
+    const {
+      deviceType,
+      getScrollingIntervalId: getScrollingId,
+      setScrollingIntervalId: setScrollingId,
+    } = useDevice();
     const [currentlyFocusedItemIndex, setCurrentlyFocusedItemIndex] = useState(0);
+    const navigator = useSpatialNavigator();
```

Comment:
The whole thing you just added should probably be extracted in a local `useWebPointerScrollLogic()` hook, for clarity. This will allow to show that these lines are only relevant for web scroll and will help with code clarity (this component is already complex enough :P )

Reply:
Fixed